### PR TITLE
Enable W&B

### PR DESCRIPTION
**Pull request by pkubiak**

### CODE REVIEW

File: universal_computation/experiment.py [hunk 1]
```diff
@@ -186,10 +185,12 @@ def accuracy_fn(preds, true, x=None):
             **exp_args,
             **kwargs,
         )
+        del config['__dict__']
+        del config['__weakref__']
```
Comment (piotrhm) [inline universal_computation/experiment.py:189]: Just out of curiosity, why are you deleting such values from config?

File: universal_computation/experiment.py [hunk 2]
```diff
@@ -186,10 +185,12 @@ def accuracy_fn(preds, true, x=None):
...
         wandb.init(
             name=f'{exp_name}-{short_name}',
             group=f'{exp_name}-{task}',
-            project=wandb_project,
+            project=exp_args['wandb_project'], entity=exp_args['wandb_entity'],
```
Comment (piotrhm) [inline universal_computation/experiment.py]: I guess newline is missing.

**Review by piotrhm — COMMENTED** [review]
Lgtm! I checked changes and everything works. 👍

File: universal_computation/experiment.py (hunk 1 above)
Comment (pkubiak) [reply universal_computation/experiment.py:189]: I am not sure :P but otherwise there were some exceptions with JSON serialization of `config` dict 🤷

Comment (piotrhm) [reply universal_computation/experiment.py:189]: I had the same exceptions when I tried to fix that issue. I thought there was some big brain time behind that fix :P Anyway thanks for the answer. 👍

**Pull request merged**